lib/kitchen/directions/bake_iframes/v1.rb

```diff
@@ -14,7 +14,7 @@ def bake(book:)
           begin
             iframe.parent.rex_link
           rescue StandardError
-            warn "Unable to find rex link for iframe with parent id=#{iframe.parent.id}"
+            warn "Unable to find rex link for iframe with parent #{iframe.parent}"
             iframe[:src]
           end
```

Comment:
the `iframe.parent`s can be changed out for just `iframe` since we're no longer id-dependent